
<launch>
    <arg name="enable_imu_trigger" default="false"/>
    <node pkg="nodelet" type="nodelet" name="vision_nodelet_right" args="manager"
          output="screen"/>
    <node pkg="nodelet" type="nodelet" name="hk_camera_right"
          args="load hk_camera/HKCameraNodelet vision_nodelet_right"
          output="screen">
        <param name="camera_name" value="hk_right"/>
        <param name="image_width" value="1440"/>
        <param name="image_height" value="1080"/>
        <param name="pixel_format" value="bgr8"/>
        <param name="camera_info_url" value="file://$(find hk_camera)/config/hk_right.yaml"/>
        <param name="camera_frame_id" value="camera_right_optical_frame"/>
        <param name="camera_sn" value="00J67375789"/>
        <param name="frame_rate" value="160"/>
        <param name="enable_imu_trigger" value="$(arg enable_imu_trigger)"/>
        <remap from="/image_rect" to="/image_rect_right" />
    </node>
    <node pkg="nodelet" type="nodelet" name="vision_nodelet_left" args="manager"
          output="screen"/>
    <node pkg="nodelet" type="nodelet" name="hk_camera_left"
          args="load hk_camera/HKCameraNodelet vision_nodelet_left"
          output="screen">
        <param name="camera_name" value="hk_left"/>
        <param name="image_width" value="1440"/>
        <param name="image_height" value="1080"/>
        <param name="pixel_format" value="bgr8"/>
        <param name="camera_info_url" value="file://$(find hk_camera)/config/hk_left.yaml"/>
        <param name="camera_frame_id" value="camera_left_optical_frame"/>
        <param name="camera_sn" value="00K01829453"/>
        <param name="frame_rate" value="160"/>
        <param name="enable_imu_trigger" value="$(arg enable_imu_trigger)"/>
        <param name="sleep_time" value="5"/>
        <remap from="/image_rect" to="/image_rect_left" />
    </node>

</launch>
